### DISCUSSION TITLE
Versioning error in the interface

### DISCUSSION THREAD

**Discussion by Amartinez8JJ** [wintercms/winter#896] [Q&A]
Hi, i'm having a problem with the versioning interface, when i try to create a versioning manually, i get the following error: "the migration code must define a seeder migration class, leave blank if you just want to update the versioning." 

But if for example, I manually create the file in a code editor in updates and I define it in the versioning, although it appears, it does not let me apply the version, but if I do a winter:up if it correctly applies the update, or if I create a table in the database interface it also works correctly, my colleagues who had the same problem simply redid the project, I do not know if it is an interface bug or a misconfiguration of mine.

**Comment by LukeTowers** [comment]
Are you using the Winter.Builder plugin?
